docs/topics/whatsnew-eap.md

```diff
@@ -80,6 +80,49 @@ These features are available in preview, and we encourage you to try them and sh
 
 [See the full list of Kotlin language design features and proposals](kotlin-language-features-and-proposals.md).
 
+### Non-local break and continue
+
+Kotlin %kotlinEapVersion% adds a preview of another long-awaited feature, an opportunity to use non-local `break` and
+`continue`. This feature expands the toolset you can use in the scope of inline functions and reduces boilerplate code
```

Comment:
I found the first part of the sentence is too hard.
Maybe we can simplify it a bit without using "toolset"?
Something like "provides/adds more flexibility within inline functions"?